test/config.js

```diff
@@ -455,6 +456,27 @@ test('should separate strings and regexes into their own blacklist arrays', func
   t.end()
 })
 
+test('should compile wildcards from string', function (t) {
+  var agent = Agent()
+  agent.start({
+    transactionIgnoreUrls: ['foo', '/str1', '/wil*card']
+  })
+
+  t.strictEqual(
+    agent._conf.transactionIgnoreUrlRegExp.length,
+    3,
+    'was everything added?'
+  )
+
+  t.strictEquals(
+    '/foo'.search(agent._conf.transactionIgnoreUrlRegExp[0]),
+    0,
+    'was leading / added automatically to "foo"'
+  )
```

Comment:
Would want to drop this test if dropping the "auto prefix with '/'" feature above.

Reply:
👍